my_project_omar/Tasks.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, Optional
+from modules import TaskCreate, Task, UrgentTask
+from uuid import UUID, uuid4
+class Task:
+    
+    def __init__(self,title,description,due_date,status="InProgress"):
+        
+        self.title=title
+        self.description=description
+        self.due_date=due_date
+        self.status=status
+    def update_details(self,title=None, description=None, due_date=None,status=None):
```

Comment:
In the `update_details` function: i prefer to accept one object that contains all the variables you need to change. 
After accepting this object: iterate over all its properties using loop _`(You do the same operations many times, do it with loop)`_

Reply:
Momen I fix this issue but I get some help from the internet then I understand it very well